apps/address-service/domains/common/constants/providers.js

```diff
@@ -2,8 +2,12 @@ const DADATA_PROVIDER = 'dadata'
 const GOOGLE_PROVIDER = 'google'
 const YANDEX_PROVIDER = 'yandex'
 
+const INJECTIONS_PROVIDER = 'injections'
```

Comment:
Looks strange…
What do you mean here by this name of injections provider?

```suggestion
const INJECTIONS_PROVIDER = 'injections'
```

Reply:
"injections provider" is an internal provider. The application inserts its data into the data got from external providers.

Reply:
Maybe to reflect it in its name, like 'localInjections`?

Reply:
According to architecture `'injections'` is an internal feature.
Actually, the provider name is needed to detect where data is received from.